styles/colors.less

```diff
@@ -19,29 +19,23 @@
 // common colors
 @sand-bg-color: var(--sand-bg-color, #000000);
 
-@sand-text-color-rgb: var(--sand-text-color-rgb, 230, 230, 230); // #e6e6e6
-@sand-text-color: rgb(@sand-text-color-rgb);
+@sand-text-color: var(--sand-text-color, rgb(var(--sand-text-color-rgb, 230, 230, 230))); // #e6e6e6
 @sand-text-sub-color: var(--sand-text-sub-color, #abaeb3);
-@sand-shadow-color-rgb: var(--sand-shadow-color-rgb, 0, 0, 0);
-@sand-shadow-color: rgb(@sand-shadow-color-rgb, 30%);
-@sand-component-text-color-rgb: var(--sand-component-text-color-rgb, 230, 230, 230); // #e6e6e6
-@sand-component-text-color: rgb(@sand-component-text-color-rgb);
-@sand-component-text-sub-color-rgb: var(--sand-component-text-sub-color-rgb, 171, 174, 179); // #abaeb3
-@sand-component-text-sub-color: rgb(@sand-component-text-sub-color-rgb);
+@sand-shadow-color: var(--sand-shadow-color, rgb(var(--sand-shadow-color-rgb, 0, 0, 0))); // #000000
+@sand-shadow-common-color: ~"color(from" @sand-shadow-color ~"srgb r g b / 0.3)";
```

Comment:
Let's discuss the naming again.

Reply:
Conclusion: @sand-shadow-base-color for @sand-shadow-color, and @sand-shadow-color for @sand-shadow-common-color as the current @sand-shadow-common-color is the basic shadow color.

Reply:
done